### ISSUE TITLE
(🐞) `dmypy inspect` says `Unknown module` with multiple `run`s

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#15794]
`test1.py`
```py
a: int
```
`test2.py`
```py
a: str
```
```
👉 dmypy run test1.py
Daemon started
Success: no issues found in 42 source files
👉 dmypy run test2.py
Success: no issues found in 1 source file
👉 dmypy run test1.py --export types
Success: no issues found in 1 source file
👉 dmypy inspect test1.py:1:1:1:1
Unknown module: test
```
